.gitignore

```diff
@@ -2,7 +2,8 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+.idea/
+todds/
```

Comment:
I left this out because I also have a `./util/todds/wrapper.py` which is also ignored under the scope of this rule. Because of this, that is why I did not add this ignore rule. Please remove this line for now, unless you have an idea for one that will block just the `./todds` found from packaging the application.